src/crush/CrushWrapper.h

```diff
@@ -59,9 +59,9 @@ class CrushWrapper {
     DEFAULT_CHOOSE_ARGS = -1
   };
 
-  std::map<int32_t, std::string> type_map; /* bucket/device type names */
-  std::map<int32_t, std::string> name_map; /* bucket/device names */
-  std::map<int32_t, std::string> rule_name_map;
+  std::map<int32_t, std::string> type_map; /* bucket/device {type, name} e.g. {{0, "osd"}, {11, "root"}} */
+  std::map<int32_t, std::string> name_map; /* bucket/device {id, name} e.g. {{-1, "default"}, {-2, "localhost"}, {0, "osd.o"}} */
```

Comment:
IMHO, the id => name relationship is is self-explained in this context. no further comments is needed.